nodes/src/nodes/ml_sklearn/code.py

```
@@ -0,0 +1,36 @@
+import pickle
+import os
+
+class Node:
+    def __init__(self):
+        # Load model safely
+        model_path = os.path.join(os.path.dirname(__file__), "model.pkl")
+        with open(model_path, "rb") as f:
+            self.model = pickle.load(f)
```

Ruff (0.15.12)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)